portals/publisher/src/main/webapp/source/src/app/components/GlobalPolicies/Policies/GlobalSpecificComponents/Policies.tsx

```diff
@@ -0,0 +1,610 @@
+     * @returns {boolean} - True if all the required fields are filled.
+     */
+    const validate = () => {
+        let isValidate = true;
+        if (name === '') {
+            Alert.error(intl.formatMessage({
+                id: 'Policy.Name.Cannot.Be.Empty',
+                defaultMessage: 'Policy name cannot be empty',
+            }));
+            isValidate = false;
+        }
+        if (description === '') {
+            Alert.error(intl.formatMessage({
+                id: 'Policy.Description.Cannot.Be.Empty',
+                defaultMessage: 'Policy description cannot be empty',
+            }));
+            isValidate = false;
+        }
+        if ((!globalLevelPolicies.request || globalLevelPolicies.request.length === 0) &&
+            (!globalLevelPolicies.response || globalLevelPolicies.response.length === 0) &&
+            (!globalLevelPolicies.fault || globalLevelPolicies.fault.length === 0)) {
+            Alert.error(intl.formatMessage({
+                id: 'Policy.Mapping.Cannot.Be.Empty',
+                defaultMessage: 'Policy mapping cannot be empty',
+            }));
+            isValidate = false;
+        }
+        return isValidate;
+    }
+
+    /**
+     * Function to save a policy mapping.
+     * Triggers if we click save button.
+     */
+    const save = () => {
+        setLoading(true);
+
+        if (validate()){
+            /**
+             * Remove UUIDs before sending to backend.
+             * If not, as backend is not expecting UUIDs, backend validation will fail.
+             */
+            const policyMapping = removeUUIDs(globalLevelPolicies);
+
+            /**
+             * Backend Call and handle the response.
+             */
+            const requestBody = {
+                "id": uuidv4(),
+                "policyMapping": policyMapping,
+                "description": description,
+                "displayName": name,
+                "appliedGatewayLabels": []
+            };
+            const promise = API.addGatewayPoliciesToFlows(requestBody);
+            promise
+                .then((response) => {
+                    setLoading(false);
+                    if (response.status === 200 || response.status === 201) {
+                        Alert.success(intl.formatMessage({
+                            id: 'Policy.Mapping.Added.Successfully',
+                            defaultMessage: 'Policy mapping added successfully',
+                        }));                 
+                        history.goBack();
+                    }
+                    else {
+                        Alert.error(intl.formatMessage({
+                            id: 'Adding.Policy.Mapping.Error',
+                            defaultMessage: 'Error occurred while adding the policy mapping',
+                        })); 
+                    }                
+                })
+                .catch((/* error */) => {
+                    // console.error(error);
```

Comment:
```suggestion
                    
```

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)